### PR TITLE
Document Webhook Examples for Atlas Notifications

### PR DESCRIPTION
**Pull request by clstokes**

### CODE REVIEW

File: consul/alerts/notification-methods.html.md [hunk 1]
```diff
@@ -80,3 +80,130 @@ To configure PagerDuty:
...
 1. Optionally, test the alert.
+
+## Webhook
+
+Notifications can be sent to custom Webhook URLs to allow for integration
+into third-party systems. An HTTP _Post_ will be with the Post _Body_
+containing the JSON payload with the notification details. Example JSON
+payloads are shown below.
+
+> Note: The Webhook URL must be accessible from Atlas.
```
Comment (pearkes) [inline consul/alerts/notification-methods.html.md]: "the public internet"

Comment (pearkes) [inline consul/alerts/notification-methods.html.md]: ah but on-prem haha, hm

**Comment by clstokes** [comment]
Do all alert types (except Consul) also have `pending`, `starting`, `started`, and `queued` as well? Any others I'm missing?

**Comment by pearkes** [comment]
cc/ @justincampbell

File: consul/alerts/notification-methods.html.md [hunk 2]
```diff
@@ -80,3 +80,130 @@ To configure PagerDuty:
 1. Enter the service API key into the "Integrations" tab of the environment tab
    of the environment in Atlas
 1. Optionally, test the alert.
+
+## Webhook
+
+Notifications can be sent to custom Webhook URLs to allow for integration
+into third-party systems. An HTTP _Post_ will be with the Post _Body_
```
Comment (justincampbell) [inline consul/alerts/notification-methods.html.md]: > will be *sent* with

or some other verb

File: consul/alerts/notification-methods.html.md [hunk 3]
```diff
@@ -80,3 +80,130 @@ To configure PagerDuty:
...
+  }
+}
+```
+
+Valid values for `status` are:
+
+```
+critical
+passing
+warning
```
Comment (justincampbell) [inline consul/alerts/notification-methods.html.md]: Maybe sort these by best to worst (or vice-versa)?

File: consul/alerts/notification-methods.html.md [hunk 4]
```diff
@@ -80,3 +80,130 @@ To configure PagerDuty:
...
+```
+
+Valid values for `status` are:
+
+```
+canceled
+errored
+finished
+starting
+```
```
Comment (justincampbell) [inline consul/alerts/notification-methods.html.md]: Sort these by the order they would occur?

File: consul/alerts/notification-methods.html.md [hunk 5]
```diff
@@ -80,3 +80,130 @@ To configure PagerDuty:
...
+compile_errored
+compile_finished
+compile_started
+job_canceled
+job_errored
+job_finished
+job_started
+
+```
```
Comment (justincampbell) [inline consul/alerts/notification-methods.html.md]: Sort these by the order they would occur?

File: consul/alerts/notification-methods.html.md [hunk 6]
```diff
@@ -80,3 +80,130 @@ To configure PagerDuty:
...
+Valid values for `status` are:
+
+```
+applied
+applying
+confirmed
+discarded
+errored
+planned
+```
```
Comment (justincampbell) [inline consul/alerts/notification-methods.html.md]: Sort these by the order they would occur?

**Comment by justincampbell** [comment]
Awesome! :+1: at your discretion

**Comment by marvinpinto** [comment]
Looks awesome, thank you for documenting this :+1:

**Comment by justincampbell** [comment]
@clstokes All statuses:

```
Application
    compile_started
    compile_finished
    compile_errored
    job_started
    job_finished
    job_canceled
    job_errored

Build
    starting
    finished
    canceled
    errored

Run
    planned
    confirmed
    discarded
    applied
    applying
    errored

Consul
    passing
    warning
    critical
```

**Comment by clstokes** [comment]
Thanks for the review @marvinpinto and @justincampbell!

**Pull request merged**